grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -390,9 +401,50 @@ class ProductService {
             if (sortColumn) order(sortColumn, sortOrder)
         }
 
-        return results
+		if (catalogsInput && !tagsInput) {
```

Comment:
I can't see the main query code in the diff view, but the catalog filter should be integrated into the query. If you're having trouble writing HQL please let me know.

Reply:
@jmiranda you can use the expand up button to see the rest of the code

Reply:
@pmuchowski Thanks. I just found "expand" after thinking to myself "it's ridiculous I can't see more of the code on the diff screen ... there's gotta be a way ... oh, there it is"